### PR TITLE
Make Wrap Paragraph to Column action constrain wrapping to selected lines

### PR DESCRIPTION
**Pull request by mikke89**
Hi, and thanks for this plugin. I made some changes, mostly to make the plugin work better for my own use cases.

- Fix applying plaintext filetypes option
- Make Wrap Paragraph to Column action constrain wrapping to selected lines

I've also made some changes to the gradle build file to make it all compile for me.

### CODE REVIEW

**Comment by abrookins** [comment]
Oh my gosh, I totally missed your PR, @mikke89! Thanks so much for opening. I saw it tonight after finally getting tired of exactly the behavior you fixed and similarly fiddling with the gradle file. A lot. 😂 

I ended up refactoring a few things in the plugin code to drop some deprecated usages that were throwing warnings. I'll compare your implementation with where I landed and see how they combine!

File: src/main/java/com/andrewbrookins/idea/wrap/ui/WrapSettingsPanel.java [hunk 1]
```diff
@@ -52,6 +52,7 @@ public void apply() {
 
         Objects.requireNonNull(settingsProvider.getState()).setColumnWidthOverride(columnWidth);
         settingsProvider.getState().setUseMinimumRaggednessAlgorithm(useMinimumRaggednessAlgorithmCheckBox.isSelected());
+        settingsProvider.getState().setPlaintextFileTypes(plaintextFileTypesField.getText());
```
Comment (abrookins) [inline src/main/java/com/andrewbrookins/idea/wrap/ui/WrapSettingsPanel.java:55]: Nice catch!

**Comment by mikke89** [comment]
No worries, please feel free to do whatever you wish with this PR. I'm just happy to see this extension continue to get support, it's really a must-have tool! :)

**Comment by abrookins** [comment]
Closing because I integrated this change into #60. Thanks again!

**Pull request closed without merging**